### PR TITLE
Finish Mobile Version: Recent works & About me page

### PR DESCRIPTION
**Pull request by sunga12**
- Added Recent Works page
- Added About Me Page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -113,3 +110,400 @@ header {
...
+  text-decoration: none;
+  line-height: 20px;
+  padding: 10px 0 10px 0;
+  text-align: center;
+  color: #fff;
+  font-weight: 500;
+  font-family: 'Inter', 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+button.project-link-btn {
```
Comment (AHP15) [inline styles.css:324]: - [ ] Kindly make sure that the alignment of the text in this button is matching the design:

 ## Your design:
s2

## Figma design

s

**Review by AHP15 — CHANGES_REQUESTED** [review]
Hi @sunga12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

soclose
## Highlights
✅ Great design
✅ No linter errors
✅ Good GitHub flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -113,3 +110,400 @@ header {
...
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 24px;
+  background-image: url(./images/Project-BG.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow: hidden;
+  height: 386px;
+  color: white;
+}
```
Comment (pinkmoon25) [inline styles.css]: - [ ] Good job making the changes requested by the previous reviewer, I think there was a bug introduced while making the changes, please see that the project card is not visible properly and the bottom of the button is cut out :point_down: 
image

I suggest loosing the fixed height property that you have set on the project cards please try to fix it :+1:

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @sunga12 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
Linter checks :heavy_check_mark: 
github flow :heavy_check_mark: 
flex and grid :heavy_check_mark: 
follows figma template :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @sunga12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linter checks :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
flex and grid :heavy_check_mark: 
follows figma template :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**